skimage/exposure/_adapthist.py

```diff
@@ -25,7 +25,7 @@
 
 
 @adapt_rgb(hsv_value)
-def equalize_adapthist(image, ntiles_x=8, ntiles_y=8, clip_limit=0.01,
+def equalize_adapthist(image, ntiles_x=None, ntiles_y=None, clip_limit=0.01,
```

Comment:
In this case we decided to err on the side of allowing positional arguments to continue to work.